Sprint-1/1-key-exercises/2-initials.js

```diff
@@ -5,7 +5,14 @@ let lastName = "Johnson";
 // Declare a variable called initials that stores the first character of each string.
 // This should produce the string "CKJ", but you must not write the characters C, K, or J in the code of your solution.
 
-let initials = ``;
+// declaring variables for each name and take out fist character
+
+let fname_initials = `${firstName.charAt(0)}`;
```

Comment:
Because `charAt` returns a string, we don't need to interpolate this into a string as well
```suggestion
let fname_initials = firstName.charAt(0);
```

Reply:
Remove string interpolate.